.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint-staged && yarn build && git add dist && git add README.md
```

Comment:
So one thing that @offgriddev and I discovered is that this works fine when we are working locally and making PRs but we run into a problem when Dependabot creates PRs. During that case, it will upgrade a dependency but we are not running the `build` which will run a `prebuild` step that's compiling the code to be added to the `dist/` folder. This is problematic because these build files are what our actions are going to be using so technically we're not actually providing new code during Dependabot PRs. This problem exists for all our custom Github Actions so we're going to have to think about how to fix this scenario. Doesn't need to be part of this PR, just wanted to raise the issue to you.

Does this make sense? I'd be happy to walk you through this over video so it's more clear.

Reply:
Yes it totally makes sense
Is there a way to make dependabot do the build before commit/push or will we have to implement this mechanism ourself in dependabot workflow ?